### ISSUE TITLE
Corrupt JPEG data: 115936 extraneous bytes before marker 

### ISSUE THREAD

**Issue by zeynepkoyun** [IntelRealSense/RealSenseID#116]
Hi @gabime 

I get the error "Corrupt JPEG data: 115936 extraneous bytes before marker" when running the viewer.py code in python and the image looks corrupted.

**Comment by gabime** [comment]
Please provide more info about os and compiler and python versions.

**Comment by zeynepkoyun** [comment]
> Lütfen işletim sistemi, derleyici ve python sürümleri hakkında daha fazla bilgi verin.

Hi @gabime ,
Ubuntu 16.04, python3.6 and the project version I am using is v0.23.0

**Comment by gabime** [comment]
Please try the rsid-cli tool to see if the problem is python specific or from the env (press ‘c’ to get some preview frames). 

Also, please use ubuntu 18 or later.

**Comment by zeynepkoyun** [comment]
Hi @gabime 
I get the error I mentioned, especially when the device starts to heat up, but when I get an error, when I look at the realsenseID application in windows, the images look normal.
As you said, let me check with "rsid-cli" in ubuntu and share the results with you.

**Comment by zeynepkoyun** [comment]
Hi @mdisg 
Did this piece of code exactly solve your problem?
When I made this change and tested it, I noticed that the recorded images were wrong, but the display does not show the wrong image, but the camera is still broken.

**Comment by mdisg** [comment]
@zeynepkoyun yes the wrong images are ignored/dropped e.g. by the client in my case Firefox or Chromium so the data is still corrupted as I mentioned in my last post.

**Comment by zeynepkoyun** [comment]
@mdisg @gabime 
I am working with ubuntu 16.04. When I get a distorted image from the camera, the image looks fine when I look at it with the windows application.
